Gruntfile.js

```diff
@@ -40,12 +40,29 @@ module.exports = function (grunt) {
 						replacement: '.fuelux-icon$1, .glyphicon$1 '
 					}
 					
+					// ,
+					// {
+					// 	pattern:  /(\.-?[_a-zA-Z]+[_a-zA-Z0-9-]*)-hover-focus\s*/g,
+					// 	replacement: '$1-hover-focus, $1:hover:focus, button:hover:focus > $1 '
+					// }
+
 					,
 					{
-						pattern:  /(\.-?[_a-zA-Z]+[_a-zA-Z0-9-]*)-hover-focus\s*/g,
-						replacement: '$1-hover-focus, $1:hover:focus, button:hover:focus > $1 '
+						pattern:  /(\.-?[_a-zA-Z]+[_a-zA-Z0-9-]*)-checked-hover-active\s*/g,
+						replacement: '$1-TOBECHECKEDHOVERACTIVE '
```

Comment:
If this does what I think it does, it needs some commentary. Well, even if it does something different than I think it does, it needs some commentary ;) .

It appears you are setting *-checked-hover-active here, only to set it _back to_ *-checked-hover-active on (new) line 88, with the addition of *:checked:active:hover as well. I think I can see why, but, this could certainly use some road-signs for future tourists...